Bookish/Views/Book/_BookTable.cshtml

```diff
@@ -4,27 +4,34 @@
     <table class="table">
         <thead>
         <tr>
-            <th scope="col">Id</th>
-            <th scope="col">Author</th>
-            <th scope="col">Title</th>
+            @foreach (var property in typeof(BookModel).GetProperties())
+            {
+                <th scope="col">@property.Name</th>
+            }
         </tr>
         </thead>
         <tbody>
         @foreach (var book in Model)

```

Comment:
This foreach block should probably be indented once more as it's nested within `<tbody>`